### PR TITLE
Improved README.md

### PR DESCRIPTION
**Pull request by jacktherock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 [![codecov](https://codecov.io/gh/Claviz/xlstream/branch/master/graph/badge.svg)](https://codecov.io/gh/Claviz/xlstream)
 ![npm](https://img.shields.io/npm/v/xlstream.svg)
 
-# xlstream
+## xlstream
```
Comment (jansivans) [inline README.md:5]: I would like this to stay a first level heading.

**Review by jansivans — CHANGES_REQUESTED** [review]

**Comment by jansivans** [comment]
Thanks!

**Comment by jacktherock** [comment]
> Thanks!



> Thanks!

> Here we go. Thank you for your suggestion .

**Comment by jacktherock** [comment]
Hello can you do me a fever, please add hacktoberfest topic to this repo and label my file as _hacktoberfest accepted_ and check [hacktoberfest](https://hacktoberfest.digitalocean.com/hacktoberfest-update) for more info. Thank you!

**Comment by jansivans** [comment]
Yeah sure I can do this, but only if you will make some valuable PR. For example, you can cover uncovered lines with tests. 

As Hacktoberfest says:

_Participating in Hacktoberfest leads to personal growth, professional opportunities, and community building. However, it all begins with meaningful contributions to open source software._

Editing few lines in README isn't a meaningful contribution and doesn't lead you to personal growth.

**Comment by jacktherock** [comment]
Then what should I do next? Can you suggest.

**Comment by jansivans** [comment]
Please read my response more carefully - you can cover uncovered lines with tests.

**Pull request merged**